### ISSUE TITLE
Adding SELU activation function to tf general_network

### ISSUE THREAD

**Issue by gaceladri** [NervanaSystems/coach#39]
Hi guys, 
I really appreciate your awesome work! I would like you to add the **selu** activation function to your awesome project. 

At the tf architecture in **general_network** line 39

    def get_activation_function(self, activation_function_string):
        activation_functions = {
            'relu': tf.nn.relu,
            'tanh': tf.nn.tanh,
            'sigmoid': tf.nn.sigmoid,
            'elu': tf.nn.elu,
            'selu': tf.nn.selu,         # <- New Self-normalizing neural networks
            'none': None
        }

The paper:
https://arxiv.org/abs/1706.02515

And the tf funct:
https://www.tensorflow.org/versions/master/api_docs/python/tf/nn/selu

Kind regards, keep going with this awesome project.
Adrian Brunetto

**Comment by itaicaspi-intel** [comment]
Thanks @gaceladri! We are glad to hear you liked Coach
Looks like an easy addition. We will add it soon.

**Comment by itaicaspi-intel** [comment]
This feature was added in commit 43821c9630543035e3d66bb9fc6b8a9cefbadc97
